docs/token_authorization_api.md

```diff
@@ -2,6 +2,7 @@
 
 TorchServe now enforces token authorization by default
 
+TorchServe introduces a new feature called token-authorization which requires the correct token to be provided when calling an API. This is a security feature which addresses the concern of unwanted API calls. This is applicable in the scenario where an unauthorized user may try to access a running TorchServe server. The default behavior is to enable this feature which creates a key file with the appropriate tokens to be used for API calls. Users can disable this feature to prevent token authorization from being required for API calls.
```

Comment:
What happens to users who start torchserve like before? What's the expected behavior and what do they need to change?

Reply:
Since the default is token auth enabled, users who start like before will be expected to provide the token for all APIs. If users don't want to use tokens then they would have to disable using one of the provided formats.